src/python/pants/backend/go/util_rules/BUILD

```diff
@@ -1,7 +1,8 @@
 # Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
-python_sources(dependencies=[":analyzer_sources"])
+python_sources(dependencies=[":analyze_package_sources", ":generate_testmain_sources"])
 python_tests(name="tests", timeout=120)
 
-resource(name="analyzer_sources", source="generate_testmain.go")
+resource(name="generate_testmain_sources", source="generate_testmain.go")
+resource(name="analyze_package_sources", source="analyze_package.go")
```

Comment:
Not a big deal, but you can have less boilerplate by using `resources` and setting `sources=["*.go"]`. The `python_sources` would depend on the `resources` target, which is an alias for both generated `resource` targets.

Reply:
Done.